### ISSUE TITLE
html labels no longer working?

### ISSUE THREAD

**Issue by christianvoigt** [aduh95/viz.js#3]
Hi, I just tested v3.0.0-beta.7 and it seems like html labels are no longer working.

Below is an example that worked in older versions and also in my own build of the sync branch. I also tested it in http://viz-js.com/.

However with the beta.7 version the label text is "n0", "n2", "n3" instead of "A", "B", "C". I also checked the [Graphviz changelog](https://gitlab.com/graphviz/graphviz/blob/master/ChangeLog), but could not discover any recent changes that could explain this. Also, as already mentioned I believe my own build did not have this issue.

```dot
digraph "Argument Map" {

rankdir = "BT";
concentrate = "false";
ratio = "auto";
size = "10,10";
graph [bgcolor = "transparent" ]  n0 [label=<<TABLE WIDTH="180" ALIGN="CENTER" BORDER="0" CELLSPACING="0"><TR><TD WIDTH="180" ALIGN="TEXT" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000"><B>B</B></FONT></TD></TR><TR><TD ALIGN="TEXT" WIDTH="180" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000">test</FONT></TD></TR></TABLE>>, shape="box",  margin="0.11,0.055", style="filled,rounded,bold", 
color="#1b9e77", fillcolor="white", labelfontcolor="white", fontcolor="#000000", type="statement-map-node"];
  n1 [label=<<TABLE WIDTH="180" ALIGN="CENTER" BORDER="0" CELLSPACING="0"><TR><TD WIDTH="180" ALIGN="TEXT" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000"><B>A</B></FONT></TD></TR><TR><TD ALIGN="TEXT" WIDTH="180" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000">test</FONT></TD></TR></TABLE>>, margin="0.11,0.055", shape="box", style="filled,rounded", fillcolor="#1b9e77", fontcolor="#000000",  type="argument-map-node"];
  n2 [label=<<TABLE WIDTH="180" ALIGN="CENTER" BORDER="0" CELLSPACING="0"><TR><TD WIDTH="180" ALIGN="TEXT" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000"><B>C</B></FONT></TD></TR><TR><TD ALIGN="TEXT" WIDTH="180" BALIGN="CENTER"><FONT FACE="arial" POINT-SIZE="10" COLOR="#000000">test</FONT></TD></TR></TABLE>>, margin="0.11,0.055", shape="box", style="filled,rounded", fillcolor="#1b9e77", fontcolor="#000000",  type="argument-map-node"];


  n0 -> n1 [type="attack", color="#ff0000"];
  n2 -> n1 [type="support", color="#00ff00"];

}
```

**Comment by aduh95** [comment]
Can confirm it's a bug. Graphviz 2.44.0 displays it correctly, it's a problem with this implementation. I will investigate on that, thanks for the report.

**Comment by igbopie** [comment]
Wow! I had that issue yesterday.

**Comment by aduh95** [comment]
I have published v3.0.0-beta.8 on npm which renders the correct label. 

This issue was coming from some configuration files inherited from the original viz.js repo – introduced in the initial commit actually – probably to workaround a bug in `emsdk` which may have been fixed since. Anyway, removing them seems to fix the issue, and hopefully it won't introduce any other bug.
